roofit/roofit/src/RooGExpModel.cxx

```diff
@@ -321,49 +352,49 @@ double RooGExpModel::evaluate() const
     //double result = 0.5*evalCerf(fsign*u,c).real() ;  // sign=-1 !
 
     if (_basisCode!=0 && basisSign==Both) result *= 2 ;
-    //cout << "1st form " << "x= " << x << " result= " << result << endl;
+    //cout << "1st form " << "x= " << x << " result= " << result << std::endl;
```

Comment:
This line could be conditional on `if (verboseEval()>...)` and similar for similar commented lines below ?

Reply:
Thanks for the suggestion, I will deal with this in the next refactoring.